src/amo/components/RatingManager/index.js

```diff
@@ -97,7 +97,7 @@ export class RatingManagerBase extends React.Component {
         return i18n.gettext('Log in to rate this dictionary');
       case ADDON_TYPE_LANG:
         return i18n.gettext('Log in to rate this language pack');
-      case ADDON_TYPE_SEARCH:
+      case ADDON_TYPE_OPENSEARCH:
         return i18n.gettext('Log in to rate this search engine');
```

Comment:
Should we call this a 'search plugin?' That's what MDN calls them but sometimes it says 'search engine plugin.'